### PR TITLE
Merge from base repo

### PR DESCRIPTION
**Pull request by jdordoigne**

### CODE REVIEW

**Comment by mssola** [comment]
@jdordoigne I think this is a mistake no ? Am I missing something ?

File: config/config-local.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  uid: "uid"
+
+  guess_email:
+    enabled: true
+    attr: "mail"
+
+  authentication:
+    enabled: true
+    bind_dn: "cn=portus,ou=services,dc=socialbakers,dc=com"
+    password: "tO2qQX4W3Er2D3xs7Fyg"
```
Comment (cyphar) [inline config/config-local.yml]: I would recommend changing this password.

**Comment by jdordoigne** [comment]
Yes sorry, bad action

**Comment by jdordoigne** [comment]
Yes sorry, mistake from my side.

**Pull request closed without merging**